### PR TITLE
RMRPA-42 Remove unused IgnoreListType

### PR DESCRIPTION
**Pull request by KUJO7**
https://re-motion.atlassian.net/browse/RMRPA-42

### CODE REVIEW

**Review by Frazerus — APPROVED** [review]

**Comment by Frazerus** [comment]
Commit message has the wrong issue number

File: Core/Configuration/Data/Config.cs [hunk 1]
```diff
@@ -33,9 +33,6 @@ public class Config
   [XmlElement("prereleaseMergeIgnoreList")]
   public IgnoreList PreReleaseMergeIgnoreList { get; set; }
 
-  [XmlElement("tagStableMergeIgnoreList")]
```
Comment (MichaelKetting) [inline Core/Configuration/Data/Config.cs:36]: @Frazerus & @KuJo7 what happens when the element is still part of the config file in a project? will it get ignored or fail and we need to fix it everywhere?

Comment (Frazerus) [reply Core/Configuration/Data/Config.cs:36]: Just tested with @KuJo7, it does not matter/will be ignored.

Comment (KUJO7) [reply Core/Configuration/Data/Config.cs:36]: As we checked , it will get ignored if it is still part of the config file

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**